src/SAML2/AuthnRequest.php

```diff
@@ -660,10 +662,8 @@ public function getRequestedAuthnContext()
      * @param array $requestedAuthnContext The RequestedAuthnContext.
      * @return void
      */
-    public function setRequestedAuthnContext($requestedAuthnContext)
+    public function setRequestedAuthnContext(array $requestedAuthnContext)
```

Comment:
This is not something we can add in the 3.x branch, right? Type hinting with scalar types is not available in PHP 5.x.

Reply:
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

Since 5.1